domains/xpr.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "xprsh site",
+  "repo": "https://codeberg.org/xpr/pages",
+  "owner": {
+    "username": "xpr",
+    "email": "xpr@atl.tools",
+    "discord": "xprsh"
+  },
+  "record": {
+    "CNAME": "xprsh.codeberg.page"
```

Comment:
I cannot react your website, 404
Codeberg says:
> "could not find a valid repository or branch for repository: "

Reply:
so do i need to add the branch in the url? y/n
or i have to use github pages?